src/main/java/frc/robot/subsystems/Feeder.java

```diff
@@ -67,6 +67,7 @@ public void periodic() {
             SmartShuffleboard.put("Feeder", "Color Sensor",  "Infrared", IR);
             SmartShuffleboard.put("Feeder", "Color Sensor", "Proximity", proximity);
             SmartShuffleboard.put("Feeder", "Color Sensor", "ObjectSeen", getPiece() == null ? "null" : getPiece().getName());
+            SmartShuffleboard.put("Diagnostics", "Color Sensor", "ObjectSeen", getPiece() == null ? "null" : getPiece().getName());
```

Comment:
Just do Piece seen since anything else is useless and we only get None if the color sensor finds a color inbetween white and orange